File: HackerRankChallenges/printElementofLinkedList.js
function printLinkedList(head) {
    while(head != null){
        console.log(head.data)
        head = head.next;
    }

}

function main() {
    const llistCount = parseInt(readLine(), 10);

    let llist = new SinglyLinkedList();

    for (let i = 0; i < llistCount; i++) {
        const llistItem = parseInt(readLine(), 10);
        llist.insertNode(llistItem);
    }

    printLinkedList(llist.head);
}

File: HackerRankChallenges/printElementofLinkedList.cs
    static void printLinkedList(SinglyLinkedListNode head) {
        while(head != null){
            Console.WriteLine(head.data);
            head = head.next;
        }

    }

    static void Main(string[] args) {
        SinglyLinkedList llist = new SinglyLinkedList();

        int llistCount = Convert.ToInt32(Console.ReadLine());

        for (int i = 0; i < llistCount; i++) {
            int llistItem = Convert.ToInt32(Console.ReadLine());
            llist.InsertNode(llistItem);
        }

        printLinkedList(llist.head);
    }
}
